src/components/AvatarGroup/AvatarGroupCounterTooltipContainer.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import Tooltip from "../Tooltip/Tooltip";
+import Dialog from "../Dialog/Dialog";
+import AvatarGroupCounterTooltipContent from "./AvatarGroupCounterTooltipContent";
+import Avatar from "../Avatar/Avatar";
+import PropTypes from "prop-types";
+
+const AvatarGroupCounterTooltipContainer = ({ children, avatars, type, className, counterTooltipCustomProps }) => {
+  const counterTooltipContentExists = !!avatars?.length;
+  const counterTooltipContentComponent = (
+    <AvatarGroupCounterTooltipContent avatars={avatars} type={type} className={className} />
+  );
+
+  if (!counterTooltipContentExists && !counterTooltipCustomProps) {
+    return children;
```

Comment:
this check is not needed no? if we implement what I wrote on top of the page and...  if we don'y have counterTooltipCustomProps we still want to present the tooltip no?